### PR TITLE
Recipe #0046—Alternative representations through rendering

### PR DESCRIPTION
**Pull request by triplingual**
This PR addresses issue #46, introducing the way to have alternative modes of the same content. Distinguished from accompanyingCanvas and placeholderCanvas.

### CODE REVIEW

**Comment by triplingual** [comment]
[Direct link to recipe](https://preview.iiif.io/cookbook/0046-rendering/recipe/0046-rendering/)
[Recipe index page](https://preview.iiif.io/cookbook/0046-rendering/)

**Comment by glenrobson** [comment]
Comments from the IIIF Cookbook Editors:

### Suggested title:
Providing Alternative Representations

### Use case
 * Could you also mention this rendering field can also be used for other things as well as PDF for example linking to TEI

### Example
* Could you add a short description on how to get to the PDF in Mirador. 
* Typo:  “program at a whole,” “program as a whole,”
* We don't think backticks are needed around Manifest and canvas in the example text.  

### Related and Restrictions
 * British spelling: multispectral flavoured example (in related and restrictions)

### Related recipes
 * Link to 3d rendering recipe in related 

The editors are happy for this to proceed to TRC without another review. 

Note they did find an issue for me to fix which is causing a http/https issue in Chrome. The info.json here:

https://iiif.io/api/image/3.0/example/reference/4f92cceb12dd53b52433425ce44308c7-ucla_bib1987273_no001_rs_001/info.json

has a http url in the `@id` field where it should be https. There is no change required in your manifest as that is correct but I need to change the Image server.

**Comment by triplingual** [comment]
+ ✅ Providing Alternative Representations (thanks @mcwhitaker for this selfsame suggestion!)
+ ✅ Could you add a short description on how to get to the PDF in Mirador.
+ ✅ Typo: “program at a whole” ==> “program as a whole”
+ ✅ We don't think backticks are needed around Manifest and canvas in the example text.
+ ✅ British spelling: multispectral flavoured example (in related and restrictions)
+ ✅ Link to 3d rendering recipe in related
    + I linked to #59 (Museum object) — Should I link to #125 (3D Models) instead? The former uses rendering also.

File: recipe/0046-rendering/index.md [hunk 1]
```diff
@@ -1,43 +1,42 @@
...
-{
-  "@context": [
-    "http://www.w3.org/ns/anno.jsonld",
-    "http://iiif.io/api/presentation/{{ page.major }}/context.json"
-  ],
-  "id": "https://example.org/iiif/book1/manifest",
-  "type": "Manifest" 
-}
-```
+To see the property in actiono in Mirador, toggle the sidebar by activating the three-line ("hamburger") menu in the upper left-hand corner of the content window. You should then, in the "Related" area, see the link under the "Alternate formats" heading.
```
Comment (zimeon) [inline recipe/0046-rendering/index.md:27]: Typo: "actiono" -> "action"

**Review by zimeon — CHANGES_REQUESTED** [review]

File: recipe/0046-rendering/index.md (hunk 1 above)
Comment (triplingual) [reply recipe/0046-rendering/index.md]: Fixed in https://github.com/IIIF/cookbook-recipes/pull/214/commits/fb5709e5ad054c31d2fe1c36656f06e251134317

**Comment by glenrobson** [comment]
Cookbook editors think this is ready to go to TRC with no further changes.

**Pull request merged**